mpi/outer_script.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+CORONA_COMPILERS="
+gcc
+clang
+intel-classic
+"
+
+CORONA_MPIS="
+mvapich2
+"
+
+export TESTS="hello
+abort
+version
+"
+
+IS_CORONA=$(echo $HOSTNAME | grep corona)
```

Comment:
In LC we have an environment variable `LCSCHEDCLUSTER` which might be more useful, e.g. on corona
```
LCSCHEDCLUSTER=corona
```
Then you can test `if test "$LCSCHEDCLUSTER" = "corona"`.

As a side note, you could generalize this using bash [indirect expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html)

> If the first character of parameter is an exclamation point (!), and parameter is not a nameref, it introduces a level of indirection. Bash uses the value formed by expanding the rest of parameter as the new parameter; this is then expanded and that value is used in the rest of the expansion, rather than the expansion of the original parameter. This is known as indirect expansion. The value is subject to tilde expansion, parameter expansion, command substitution, and arithmetic expansion. If parameter is a nameref, this expands to the name of the variable referenced by parameter instead of performing the complete indirect expansion. The exceptions to this are the expansions of ${!prefix*} and ${!name[@]} described below. The exclamation point must immediately follow the left brace in order to introduce indirection. 


E.g.
```bash
#!/bin/sh

corona_COMPILERS="
gcc
clang
intel
"

corona_MPIS="
mvapich2
"
MPIS="${LCSCHEDCLUSTER}_MPIS"
COMPILERS="${LCSCHEDCLUSTER}_COMPILERS"

for mpi in ${!MPIS}; do
    for compiler in ${!COMPILERS}; do
        echo "$compiler $mpi"
    done
done
```

```console
$ bash test.sh
gcc mvapich2
clang mvapich2
intel mvapich2
```

Reply:
Thanks @grondo for the tip! I think I've addressed all of your first-round comments. This is ready for a second review.